### PR TITLE
Trim Voilà hype

### PR DESCRIPTION
**Pull request by palewire**
Looking to tighten up this tag line a bit.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -170,7 +170,7 @@ <h4>Code meets data</h4>
                     <div class="col-md-6 nb-highlight-text">
                         <h3>Communicate your results with Voilà.</h3>
                         <h4 class="nb-desc">
-                            Voilà helps you communicate insights, by transforming a Jupyter Notebook into a stand-alone web application you can share. It gives you control over what your readers experience in a secure and customizable interactive dashboard.
+                            Voilà helps communicate insights by transforming notebooks into a secure, stand-alone web application that you can customize and share.
```
Comment (afshin) [inline index.html:173]: I think this language is snappier, thanks!

I would suggest that "notebooks" and "web applications" should match in number.

```suggestion
                            Voilà helps communicate insights by transforming notebooks into secure, stand-alone web applications that you can customize and share.
```

Comment (palewire) [reply index.html]: Thanks, I accepted your proposal.

**Review by afshin — APPROVED** [review]

**Review by Carreau — APPROVED** [review]

**Comment by Carreau** [comment]
Approved, but I'd like @SylvainCorlay +1.

**Review by SylvainCorlay — APPROVED** [review]

**Comment by meeseeksmachine** [comment]
This pull request has been mentioned on **Jupyter Community Forum**. There might be relevant details there:

https://discourse.jupyter.org/t/governance-office-hours-meeting-minutes/1480/170

**Pull request merged**